### ISSUE TITLE
zef should store cached repositories in XDG_CACHE_HOME on Linux

### ISSUE THREAD

**Issue by samcv** [ugexe/zef#158]
On Linux the proper folder to put the cached repositories is ~/.cache, or XDG_CACHE_HOME if it defined and non-empty.

Config files should be stored in ~/.config or XDG_CONFIG_HOME if defined and non-empty.

~/.cache/zef or ~/.config/zef would be neater and allow us to separate the cache files from the configuration files as well as fitting with the XDG 

Thoughts on this?

**Comment by ugexe** [comment]
Yes, zef should follow all the items in https://specifications.freedesktop.org/basedir-spec/basedir-spec-0.6.html

...but shouldn't rakudo do this as well?

**Comment by samcv** [comment]
@ugexe what is the difference between ~/.zef/tmp and ~/.zef/store ?

It would seem .zef/tmp would likely go in ~/.cache/zef but is the `store` folder also cache information or is it not?

**Comment by ugexe** [comment]
They are essentially the same thing. tmp/ is where things get fetched to initially, and then they get extracted/checked-out to store/. So tmp/ is meant to be something that can be nuked whenever, and store/ is essentially a CompUnit::Repository so nuking it will affect other things (in this case just if zef can fetch from the cache or not).

**Comment by ugexe** [comment]
https://github.com/ugexe/zef/issues/179
